eureka-core/src/main/java/com/netflix/eureka/PeerAwareInstanceRegistry.java

```diff
@@ -1,10 +0,0 @@
-package com.netflix.eureka;
-
-import com.netflix.appinfo.InstanceInfo;
-
-/**
- * @author Tomasz Bak
- */
-public interface PeerAwareInstanceRegistry extends InstanceRegistry {
```

Comment:
These abstractions were introduced for sake of unit testing, so we do not have to instantiate concrete objects. I think there is value in keeping those.